### PR TITLE
Add support for a "selected" day

### PR DESCRIPTION
**Pull request by MadAnd**
Added option to set initially selected day.
If the user clicks on any other day, that day will be set "selected", while the previously selected one will be unset.

### CODE REVIEW

File: src/clndr.js [hunk 1]
```diff
@@ -586,19 +604,7 @@
...
-      if(classNameIndex !== 0) {
-        // our unique identifier is always 23 characters long.
-        // If this feels a little wonky, that's probably because it is.
-        // Open to suggestions on how to improve this guy.
-        dateString = currentTarget.className.substring(classNameIndex + 13, classNameIndex + 23);
-        target.date = moment(dateString);
-      } else {
-        target.date = null;
-      }
+      var dateString = target.date = this.getTargetDate(currentTarget);
```
Comment (mikegioia) [inline src/clndr.js]: I'm going to remove dateString here as I don't think it's needed here now that the function was moved out.

**Comment by MadAnd** [comment]
@mikegioia I removed that variable.

File: src/clndr.js [hunk 2]
```diff
@@ -612,7 +618,7 @@
         } else {
           target.events = $.makeArray( $(this.options.events).filter( function() {
             // filter the dates down to the ones that match.
-            return this._clndrDateObject.format('YYYY-MM-DD') == dateString;
+            return this._clndrDateObject.format('YYYY-MM-DD') == target.date;
```
Comment (mikegioia) [inline src/clndr.js]: I think the target.date is a moment object and would need a similar .format() call?

**Comment by mikegioia** [comment]
@MadAnd I'm going to modify this, merge and commit. That `getTargetDate` method can just return a dateString and stay compliant.

**Pull request merged**